tutorials/2d/using_tilemaps.rst

```diff
@@ -6,178 +6,162 @@ Using tilemaps
 Introduction
 ------------
 
-Tilemaps are a simple and quick way to make 2D game levels. Basically,
-you start with a bunch of reference tiles (or pieces) that can be put on a
-grid, as many times each as desired - think of it like a map editor:
-
-.. image:: img/tilemap.png
+A tilemap is a grid of tiles used to create a game's layout. There are several
```

Comment:
This was updated in 2c645c823 for 3.1, so I assume it's good for 3.1.1-stable (CC @cbscribe).